near/src/config.rs

```diff
@@ -675,10 +708,20 @@ fn state_records_account_with_key(
 }
 
 /// Official TestNet configuration.
-pub fn testnet_genesis() -> GenesisConfig {
-    let genesis_config_bytes = include_bytes!("../res/testnet.json");
+pub fn testnet_genesis(genesis_records: &str, genesis_config: &str) -> GenesisConfig {
```

Comment:
It seems that this function is quite generic now, so we should not call it `testnet_genesis` anymore, right?